okhttp/src/test/java/io/grpc/transport/okhttp/OkHttpClientTransportTest.java

```diff
@@ -348,7 +359,7 @@ public void windowUpdateWithInboundFlowControl() throws Exception {
     verify(frameWriter).rstStream(eq(3), eq(ErrorCode.CANCEL));
     listener.waitUntilStreamClosed();
     assertEquals(OkHttpClientTransport.toGrpcStatus(ErrorCode.CANCEL).getCode(),
-        listener.status.getCode());
+                 listener.status.getCode());
```

Comment:
I'll try to clean up the diff cheese introduced. This was IntelliJ deciding to auto-format this way after I imported the [Google code style settings](https://google-styleguide.googlecode.com/svn/trunk/intellij-java-google-style.xml).